### PR TITLE
fix: various sentinel container errors on full nodes

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

#270 Sentinel should not be started on full nodes, and should not be included in status output in `status:core` if it can never return a good status.

## What was done?

- Moved sentinel into a separate compose file
- Put conditions around sentinel status checks, so they only run on masternodes

## How Has This Been Tested?

Locally on testnet

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: src/commands/status/services.js [hunk 1]
```diff
@@ -21,9 +21,14 @@ class ServicesStatusCommand extends ConfigBaseCommand {
   ) {
     const serviceHumanNames = {
       core: 'Core',
-      sentinel: 'Sentinel',
     };
 
+    if (config.options.core.masternode.enable) {
```
Comment (shumkov) [inline src/commands/status/services.js]: Use get function

File: src/commands/status/core.js [hunk 2]
```diff
@@ -158,10 +164,12 @@ class CoreStatusCommand extends ConfigBaseCommand {
       blocks = chalk.red(coreBlocks);
     }
 
-    if (sentinelState === '') {
-      sentinelState = chalk.green('No errors');
-    } else {
-      sentinelState = chalk.red(sentinelState);
+    if (config.options.core.masternode.enable) {
```
Comment (shumkov) [inline src/commands/status/core.js]: Use get function

**Review by shumkov — CHANGES_REQUESTED** [review]

**Comment by strophy** [comment]
Thanks, I had been using this notation everywhere in the status command, so now it is fixed everywhere 🚀

**Review by shumkov — APPROVED** [review]
LGTM! 👍

**Pull request merged**